libs/rest/generated-rest-client/src/main/kotlin/net/corda/restclient/CordaRestClient.kt

```diff
@@ -0,0 +1,255 @@
+            configurationClient: ConfigurationApi? = null,
+            cpiClient: CPIApi? = null,
+            flowInfoClient: FlowInfoApi? = null,
+            flowManagementClient: FlowManagementApi? = null,
+            helloRestClient: HelloRestApi? = null,
+            hsmClient: HSMApi? = null,
+            keyManagementClient: KeyManagementApi? = null,
+            keyRotationClient: KeyRotationApi? = null,
+            memberLookupClient: MemberLookupApi? = null,
+            memberRegistrationClient: MemberRegistrationApi? = null,
+            mgmAdminClient: MGMAdminApi? = null,
+            mgmClient: MGMApi? = null,
+            networkClient: NetworkApi? = null,
+            rbacPermissionClient: RBACPermissionApi? = null,
+            rbacRoleClient: RBACRoleApi? = null,
+            rbacUserClient: RBACUserApi? = null,
+            virtualNodeClient: VirtualNodeApi? = null,
+            virtualNodeMaintenanceClient: VirtualNodeMaintenanceApi? = null
+
+        ): CordaRestClient {
+            return createHttpClient(
+                baseUrl = baseUrl.toString(),
+                username = username,
+                password = password,
+                insecure = insecure,
+                certificatesClient = certificatesClient,
+                configurationClient = configurationClient,
+                cpiClient = cpiClient,
+                flowInfoClient = flowInfoClient,
+                flowManagementClient = flowManagementClient,
+                helloRestClient = helloRestClient,
+                hsmClient = hsmClient,
+                keyManagementClient = keyManagementClient,
+                keyRotationClient = keyRotationClient,
+                memberLookupClient = memberLookupClient,
+                memberRegistrationClient = memberRegistrationClient,
+                mgmAdminClient = mgmAdminClient,
+                mgmClient = mgmClient,
+                networkClient = networkClient,
+                rbacPermissionClient = rbacPermissionClient,
+                rbacRoleClient = rbacRoleClient,
+                rbacUserClient = rbacUserClient,
+                virtualNodeClient = virtualNodeClient,
+                virtualNodeMaintenanceClient = virtualNodeMaintenanceClient
+            )
+        }
+
+        // Prepare insecure settings for SSL
+        private fun prepareInsecureSettings(): Pair<Array<TrustManager>, SSLContext> {
+            val trustAllCerts: Array<TrustManager> = arrayOf(
+                object : X509TrustManager {
+                    override fun checkClientTrusted(chain: Array<X509Certificate>, authType: String) {}
+                    override fun checkServerTrusted(chain: Array<X509Certificate>, authType: String) {}
+                    override fun getAcceptedIssuers(): Array<X509Certificate> {
+                        return arrayOf()
+                    }
+                }
+            )
+
+            val sslContext = SSLContext.getInstance("SSL")
```

Comment:
```suggestion
            val sslContext = SSLContext.getInstance("TLS")
```
This *might* resolve SonarCloud violation.